storage/perfschema/pfs_atomic.h

```diff
@@ -1,141 +0,0 @@
-/* Copyright (c) 2009, 2011, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I think that unused shortcuts from my_atomic.h can also be removed (but the header itself should remain as a meta-header)

Reply:
I also looked into this a bit when working on this jira, and found that it would affect other components (`my_open.c` and `my_fopen.c` for example). I surely would like to do that but perhaps in another PR?